ducks/loans.ts

```diff
@@ -55,7 +54,7 @@ export default function reducer(state: LoansState = initialState, action: AnyAct
 }
 
 // hardcoded root just for testing - will be removed in next pr
-export function loadLoans(tinlake: Tinlake): ThunkAction<Promise<void>, LoansState, undefined, Action> {
+export function loadLoans(tinlake: any): ThunkAction<Promise<void>, LoansState, undefined, Action> {
```

Comment:
What happened to typing here?

Reply:
`tinlake.contractAddresses` doesn't seem to be exported somehow (or at least i couldn't figure out how to access it)